### DISCUSSION TITLE
Issue with PDF Writer on upgrade to 3.12.0

### DISCUSSION THREAD

**Discussion by snceaston** [nextgenhealthcare/connect#4790] [General]
Just noted that since upgrading to 3.12.0 we are not getting any images included in PDFs.  

It seems that it does not like `<img src="/data/graphics/ESTHLOGO.png"></img>` (for example) and requires `<img src="file:///data/graphics/ESTHLOGO.png"></img>` (you have to specify file URI).  

Hope this helps others.

**Comment by cturczynskyj** [comment]
I found [this closed issue](https://github.com/danfickle/openhtmltopdf/issues/573) in the GitHub issues of the library we are using and it seems to confirm that you need to use a proper URI. I'll add a note to the 3.12.0 upgrade guide. 

Sorry about that, and thank you for bringing it to our attention.

**Reply by snceaston** [reply]
Thanks Chris.  We had been looking specifically for issues with PDF creation since we knew that they library had changed from iText.
